src/typing/flow_js.ml

```diff
@@ -3696,6 +3696,15 @@ let rec __flow cx ((l: Type.t), (u: Type.use_t)) trace =
       ) in
       rec_flow_t cx trace (ArrT (reason, f t, List.map f ts), tout)
 
+    | _, TupleMapT (reason, funt, tout) ->
```

Comment:
I wonder if this wildcard lower bound interacts badly with the assumptions made [here](https://github.com/facebook/flow/blob/master/src/typing/flow_js.ml#L2821-L2832).